Ray.jl/src/object_store.jl

```diff
@@ -20,6 +22,7 @@ if run in an `@async` task.
 If the task that generated the `ObjectID` failed with a Julia exception, the
 captured exception will be thrown on `get`.
 """
+get(obj_ref::ObjectRef) = _get(take!(ray_jll.get(convert(ray_jll.ObjectID, obj_ref.oid))))
```

Comment:
what's the point of the `convert` in there?  should be an `ObjectIDAllocated <: ObjectID` already right?

Reply:
Whoops, I made a mistake here. This was intended to be:
```suggestion
get(obj_ref::ObjectRef) = _get(take!(ray_jll.get(convert(ray_jll.ObjectID, obj_ref))))
```
Felt a bit odd doing the field access here but either should work. What's your preference?

Reply:
I think field access is fine!  This is defining the translation layer so it's fine IMO to access the internal fields, right?

Reply:
the `_get(take!(get` turducken is a little funny but I understand the reason for the different levels of indirection